docs/customization/translation.rst

```diff
@@ -0,0 +1,51 @@
+Customizing Translations
+========================
+
+.. note::
+
+    We've adopted a convention of overriding translations in the ``AppBundle\Resources\translations`` directory.
+
+Why would you customize a translation?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you would like to change any of the translation keys defined in Sylius in any desired language.
+
+For example:
+
+* change "Last name" into "Surname"
+* change "Add to cart" into "Buy"
+
+and many other places where you can customize the text content of pages.
+
+How to customize a translation?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to customize a translation in your project:
+
+1. If you don't have create the ``AppBundle\Resources\translations\messages.en.yml`` for english translations.
+
+.. note::
+
+    You can create different files for different locales (languages). For example ``messages.pl.yml`` will hold only polish translations,
```

Comment:
Still `will` => `should`. It will hold whatever someone puts in it, especially if it's created by hand, but it should contain only polish translations.